packages/shopify-cart/README.md

```diff
@@ -231,7 +231,8 @@ const { cart, userErrors, errors } = await cartClient.cartLinesAdd({
   cartId: 'gid://shopify/Cart/112233',
   lines: [
     {
-      id: 'gid://shopify/CartLine/e543caf96fa645b54d07ab2035ef9dba?cart=a7aad2fb1e6611422c946f6827710550',
+      cartLineId: 'gid://shopify/CartLine/e543caf96fa645b54d07ab2035ef9dba?cart=a7aad2fb1e6611422c946f6827710550'
```

Comment:
`cartLineId` is not a property of `NacelleCartLineItemInput`.

```suggestion
```

Reply:
> We might also want to adjust the lines where we link out to Shopify since our inputs no longer 1:1 match Shopify's
> 
> eg:
> 
> > For more information on the cart optional values, checkout Shopify's [`CartInput`][shopify-cart-input]

Good catch @mdarrik thank you!

Reply:
@KrisQ thanks for updating the method name. But the suggestion still stands - `cartLineId` is not a property of `lines` (`NacelleCartLineItemUpdateInput`) in `cartLinesUpdate` either:

![cartLinesUpdate line params id attributes nacelleEntryId quantity sellingPlanId](https://user-images.githubusercontent.com/5732000/191350061-88e45943-0b02-4361-85aa-9862e84ff234.png)

Reply:
Apologies for the confusion, It should be `id` you're right

Reply:
No worries at all!